config/backend_connector.go

```diff
@@ -41,7 +41,7 @@ var (
 )
 
 type BackendConnector interface {
-	Init(url string, l logrus.FieldLogger) error
+	Init(url string, l logrus.FieldLogger, opts ...ConnectorOptions) error
```

Comment:
ping @someone1 fyi

Reply:
Should be exported now. I was completely unaware of the plugin use case, exported options should be generic enough for any type of connector, not just SQL (e.g. redis, mongo, etc).